scripts/migrate_actions_settings_expressions.js

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const pathLib = require('path');
+const program = require('commander');
+
+const stack_version = program.stack || require('../lerna.json').version;
+
+program
+	.version(stack_version)
+	.option('-d, --debug', 'display more info')
+	.option('-q, --quiet', 'do not display logs')
+	.option('-p, --path [value]', 'path of the settings.json to update')
+	.option('-f, --folder [value]', 'path of the folder containain json to update');
+
+program.on('--help', () => {
+	console.log('To update your project settings : ');
+	console.log('>node bin/version.js --path ../yourapp/settings.json');
+	console.log('If your project use a folder with multiple json');
+	console.log('>node version.js --folder ../yourapp/settings');
+});
+
+program.parse(process.argv);
+
+if (!program.path && !program.folder) {
+	throw new Error('You must path either -p or -f');
+}
+
+if (program.debug) {
+	console.log(`use stack version ${stack_version}`);
+}
+
+
+const JSONs = [];
+
+function readFolderContent(folder) {
+	console.log('scan ', folder)
+	fs.readdirSync(folder).forEach((path) => {
+		const fullpath = pathLib.join(folder, path);
+		if (path.endsWith('.json')) {
+			JSONs.push(fullpath);
+		} else {
+			readFolderContent(fullpath);
+		}
+	});
+	console.log('found ', JSONs);
+}
+
+if (program.path) {
+	JSONs.push(program.path);
+} else if (program.folder) {
+	readFolderContent(program.folder);
+}
+
+const ATTRS = ['active', 'available', 'disabled', 'inProgress'];
+
+/**
+ * from version XX to version YY we have introduce the following change:
+ * action.available = 'myExpression' was evaluated using cmf expression
+ * @param {Object} settings
+ */
+function getActions(settings) {
+	if (!settings || !settings.actions) {
+		return {};
+	}
+	const actions = {};
+	Object.keys(settings.actions).map(key => {
```

Comment:
`.map()` --> `.forEach()`, you don't map anything, but process something and insert in a new object